README.md

```diff
@@ -1,30 +1,24 @@
-# Pants Build System
+# Pantsbuild
```

Comment:
To me, as a headline, `Pants Build System` read better. That also leaves a hint that the project name is `Pants`, but we interchangeably refer to it as `Pantsbuild` as well (as Chris comment also was onto leaving a aka hint of some sort).